XSIAM | Add Vectra Event Collector (#24824)
demisto/content: Packs/Vectra_AI/Integrations/VectraAIEventCollector/VectraAIEventCollector_description.md

## Vectra Event Collector Help

### Retrieve API Token

> To authenticate this integration with Vectra, we need to generate an API token in Vectra Dashboard. 

Reviewer: ```suggestion
To authenticate this integration with Vectra, you need to generate an API token in Vectra Dashboard. 
```

> Every user created on the Vectra platform has access to his or her API token under the **My Profile** page. 

Reviewer: ```suggestion
Every user created on the Vectra platform has access to their API token under the **My Profile** page. 
```
